### PR TITLE
Python 2 to Python 3

### PR DESCRIPTION
**Pull request by pomme-abricot**
Only modernize

### CODE REVIEW

File: src/openalea/strawberry/analysis.py [hunk 1]
```diff
@@ -22,7 +29,7 @@ def to_dataframe(g, vertices=[], f=None):
     d = dict()
     for v in vertices:
         props_v = f(v)
-        for k, value in props_v.iteritems():
+        for k, value in six.iteritems(props_v):
```
Comment (pradal) [inline src/openalea/strawberry/analysis.py:32]: replce iteritems by items

**Comment by pomme-abricot** [comment]
@pradal @mlabadie can you review and merge my PR.

File: src/openalea/strawberry/Import_mtgfile.py [hunk 2]
```diff
@@ -40,5 +42,5 @@ def plant_number_by_varieties(g):
     genotype = set(g.property("Genotype").values())
 
     for geno in genotype:
-        no_plants= g.property("Genotype").values().count(geno)
-        print geno, ":", no_plants, "plants"
\ No newline at end of file
+        no_plants= list(g.property("Genotype").values()).count(geno)
+        print((geno, ":", no_plants, "plants"))
```
Comment (pradal) [inline src/openalea/strawberry/Import_mtgfile.py:46]: Do not print inside the code. use, if needed logging instead

File: src/openalea/strawberry/Visualization.py [hunk 3]
```diff
@@ -84,47 +89,47 @@ def plant_positions(g, by=['Genotype'], vids=[]):
...
     else:
-        vids = [g.component_roots_at_scale_iter(vid, scale=max_scale).next() for k, d in my_property.iteritems() for k2, v in d.iteritems() for vid in v]
+        vids = [next(g.component_roots_at_scale_iter(vid, scale=max_scale)) for k, d in my_property.items() for k2, v in d.items() for vid in v]
 
 
     positions = []
     x, y = x0, y0
     for genotype in my_property:
-        print genotype
+        print(genotype)
```
Comment (pradal) [inline src/openalea/strawberry/Visualization.py:132]: Idem: remove or log print statements

File: src/openalea/strawberry/Visualization.py [hunk 4]
```diff
@@ -134,7 +139,7 @@ def plant_positions(g, by=['Genotype'], vids=[]):
             y = y0
         else:
             for name2 in my_property[genotype]:
-                print name2
+                print(name2)
```
Comment (pradal) [inline src/openalea/strawberry/Visualization.py:142]: idem.

**Review by pradal — COMMENTED** [review]
Just remove unneeded print statements

**Review by pradal — APPROVED** [review]

**Pull request merged**